test/install_conda_packages_test.Dockerfile

```diff
@@ -38,7 +38,7 @@ COPY packages /packages
 
 FROM pre_install AS test_python
 ARG CONDA_CLEAN="conda clean --quiet --yes --all"
-ARG MAMBA_INSTALL="mamba install --quiet --yes --channel katanagraph --channel /packages"
+ARG MAMBA_INSTALL="mamba install --quiet --yes --channel katanagraph --channel /packages -c katanagraph/label/test -c defaults"
```

Comment:
I don't want to include defaults here because in the past I have had confusing problems when mixing anaconda and conda-forge packages. Why did you add this? I'd like to try to figure out another solution for you if possible.

Reply:
This actually must have come from jason's branch which I was tracking. I don't actually know if it is necessary.